pages/index.tsx

```diff
@@ -7,8 +7,8 @@ const Home: React.FC<unknown> = () => {
   return (
     <Layout>
       <Head>
-        <title>Electify</title>
-        <link rel="icon" href="/favicon.ico" />
+        <title>Home | Electify</title>
```

Comment:
We don't need the Home explicitly on the Homepage, can you change it to just **Electify**

Reply:
Sure.